server/routes/index.js

```diff
@@ -3,7 +3,7 @@ var router = express.Router();
 
 /* GET home page. */
 router.get('/', function (req, res, next) {
-  res.send(`<form action="/pay/payment" method="post">
+  res.send(`<form action="/pay/payment/123" method="post">
```

Comment:
Is the URL supposed to be /pay/payment/123 or is that temporary

Reply:
If it's something that will eventually be changed in the future, add in a TODO comment.
```
// TODO: replace route with proper form action
```